packages/calling-component-bindings/src/providers/CallProvider.tsx

```diff
@@ -57,5 +60,26 @@ export const CallProvider = (props: CallProviderProps): JSX.Element => <CallProv
  * @public
  */
 export const useCall = (): Call | undefined => {
-  return useContext(CallContext)?.call;
+  const call = useContext(CallContext)?.call;
+  if (call && !isACSCall(call)) {
+    throw new Error('TeamsCall object was provided, try useTeamsCall() instead');
```

Comment:
Future proofing: We're not sure here that it's a `TeamsCall`, only that it is not an ACS call. Let's reword this to `'Incorrect call type: must provide a `Call` object.`.